src/UIKit/UIView+Geometry.swift

```diff
@@ -0,0 +1,69 @@
+//
+//  UIView+Geometry.swift
+//
+//  Created by Logan Wright on 10/24/15.
+//  Copyright © 2015 lowriDevs. All rights reserved.
+//
+
+public extension UIView {
```

Comment:
i feel like these methods play a lot nicer as cgrect extensions than uiview extensions.

then 
view.frame.width
view.bounds.halfwidth

just works without us having to define separate methods. and i think it adds some readability.